draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -374,6 +374,39 @@ Note: When using this profile with other cryptosuites, it is recommended to be e
 
 `iat` and `exp` JWT claims express both the validity period of both the signature and the claims about the subject, unless there is a separate claim used to express the validity of the claims.
 
+# Security Considerations
+
+TBD.
+
+## Media Type Registration
+
+This section requests registration of the following media types [@RFC2046] in
+the ["Media Types" registry](https://www.iana.org/assignments/media-types/media-types.xhtml#application) in the manner described
+in [@RFC6838].
+
+To indicate that the content of a JWS is a Wallet Instance Attestation:
+
+  * Type name: application
+  * Subtype name: wallet-attestation+jwt
+  * Required parameters: n/a
+  * Optional parameters: n/a
+  * Encoding considerations: binary; A JWT-based Wallet Instance Attestation object is a JWT; JWT values are encoded as a series of base64url-encoded values (some of which may be the empty string) separated by period ('.') characters.
+  * Security considerations: See (#security-considerations) of [[ this specification ]]
```

Comment:
this attestation is defined in Section {#wallet-attestation-schema}, security considerations related to the attestation should be specified there

Reply:
I just have changed the link/ref to #wallet-attestation-schema